guide/payments/request.md

```diff
@@ -0,0 +1,477 @@
+   caption = "Contact cards can store the users reusable payment requests."
+   alt-text = "Screen showing users contact card."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/contact-card.png"
+   retina = "/assets/images/guide/payments/request/contact-card@2x.png"
+   layout = "shadow"
+   caption = "Contact cards can store reusable payment requests of other users too."
+   alt-text = "Screen showing another users contact card."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+## Withdrawal request
+
+Another way a user can request bitcoin is from a withdrawal request. A withdrawal request is a payment request that lets the receiver withdraw bitcoin from another user's wallet. Unlike previously mentioned payment requests, withdrawal requests are generated by the sender. These can be useful for merchants providing refunds to customers.
+
+We explore generating a withdrawal request in our [sending bitcoin]({{ "/guide/payments/send/" | relative_url }}) section. Wallets that do not support withdrawal requests should provide users with a human readable error if an attempt is made to use one. Our [payments format page]({{ "/guide/payments/send/payment-request-formats/" | relative_url }}) goes into more detail on withdrawal requests.
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/withdrawal-qr.png"
+   retina = "/assets/images/guide/payments/request/withdrawal-qr@2x.png"
+   layout = "shadow"
+   caption = "A withdrawal request is another way a user can request bitcoin."
+   alt-text = "Screen showing withdraw request QR code."
+   width = 250
+   height = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/payments/request/withdraw-request.png"
+   retina = "/assets/images/guide/payments/request/withdraw-request@2x.png"
+   layout = "shadow"
+   caption = "With withdrawal requests the user is “pulling” bitcoin from another users wallet."
+   alt-text = "Screen showing withdraw request UI."
+   width = 250
+   height = 541
+%}
+
+</div>
+
+The more payment request formats a wallet supports generating and receiving to, the more interoperable it will be. Some wallets may not support sending to a particular format so having several options makes it easier to request bitcoin.
+
+## Sharing payment requests
+
+Once a user has generated a payment request the next action to take is sharing it with the sender. There are several ways in which this can be done. Supporting all methods of sharing across payment formats makes it easier for users to request bitcoin.
+
+### Plaintext
+
+Sharing a payment request as plaintext involves simply copying and sharing it with a sender. It is typical for Bitcoin wallets to provide a share button that allows receivers to share a payment request in this manner conveniently. Both reusable and single-use requests should be shareable as plaintext.
+
+### QR code
```

Comment:
How about moving `QR code` below `Payment link`, as the `QR code` is just a visual representation of `Plaintext` and `Payment link`?